### ISSUE TITLE
Date fields have no date picker

### ISSUE THREAD

**Issue by ebonweaver** [snipe/snipe-it#3644]
Expectation: clicking the calendar icon next to a date field would pop up a calendar picker

Actual: the icon isn't a button, it serves no apparent purpose

Version 3.6.2
Confirmed in Demo

Call it a feature request I guess, but unclear why the icon is there when it does nothing, so gives the impression there is a bug where the date picker popup is broken?

**Comment by snipe** [comment]
Which page(s) are you on when you're seeing this? The datepicker must have broken somehow.

**Comment by ebonweaver** [comment]
I'm no coder so I can't tell what was changed in that post.  For completeness and to address your question, we noticed three things related to this, hopefully they have both been addressed in some way:
1. The icon with the calendar does nothing, anywhere in the system.  Expectation would be that pops up the date picker modal.
2. Clicking in the text field for a built in date field pops up the modal picker.  Expectation would be it does not, but rather allows free typing as compared to the icon being the picker as mentioned in #1.
3. Clicking in the text field for a custom date field just allows you to type, as expected, but is a behavior inconsistent with #2, so worth mentioning.
Thanks!
